fetch.js

```diff
@@ -193,6 +194,8 @@
         this._bodyBlob = body
       } else if (support.formData && FormData.prototype.isPrototypeOf(body)) {
         this._bodyFormData = body
+      } else if (support.searchParams && URLSearchParams.prototype.isPrototypeOf(body)) {
```

Comment:
Maybe not. To be sure, one could perform search params serialization manually via `toString()` when passing it to `fetch()`.